<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StartScreen</class>
 <widget class="QMainWindow" name="StartScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartScreen</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="mainVBL">
      <item>
       <widget class="QWidget" name="startScreenWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="startScreenVBL" stretch="0,2,2,3,0">
           <item>
            <widget class="QLabel" name="titleL">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>100</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>50</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>Dots Game</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="numOfPlayersHBL" stretch="1,2">
             <item>
              <widget class="QLabel" name="numberOfPlayersL">
               <property name="font">
                <font>
                 <pointsize>45</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Number of Players:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="numberOfPlayersCB">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>75</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>40</pointsize>
                </font>
               </property>
               <property name="layoutDirection">
                <enum>Qt::LeftToRight</enum>
               </property>
               <item>
                <property name="text">
                 <string>2 Players</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>3 Players</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4 Players</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="boardSizeHBL" stretch="1,2">
             <item>
              <widget class="QLabel" name="boardSizeL">
               <property name="font">
                <font>
                 <pointsize>45</pointsize>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Board Size:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="boardSizeCB">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>75</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>40</pointsize>
                </font>
               </property>
               <item>
                <property name="text">
                 <string>4x4</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>6x6</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>8x8</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="startB">
             <property name="font">
              <font>
               <pointsize>40</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::Maximum</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuNew_Game">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionNew_Game"/>
   </widget>
   <addaction name="menuNew_Game"/>
  </widget>
  <action name="actionNew_Game">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
